src/v/kafka/server/tests/CMakeLists.txt

```diff
@@ -106,6 +106,20 @@ rp_test(
   LABELS kafka
 )
 
+rp_test(
+  BENCHMARK_TEST
+  BINARY_NAME kafka_fetch
+  SOURCES fetch_bench.cc
+  LIBRARIES 
+    Seastar::seastar_perf_testing 
+    Boost::unit_test_framework 
+    v::application 
+    v::kafka_test_utils
+  # the args below are just to keep it fast
+  ARGS "-c 1 --duration=1 --runs=1 --memory=4G"
```

Comment:
What does "fast" mean? I feel like we shouldn't change duration and runs?

Reply:
Yeah, good point this was a copy & paste from a different microbenchmark. But its worth discussing what these values should reasonably be. I'll do a bunch of runs locally and see what values for these reduce variance the most. Assuming any change from the default is even required.